jenkins-scripts/lib/windows_library.bat

```diff
@@ -149,7 +149,8 @@ git clone https://github.com/ignition-tooling/gazebodistro %gzdistro_dir% -b %GA
 :: Check if ci_matching_branch name is used
 if "%ghprbSourceBranch%" == "" (echo ghprbSourceBranch is unset) else (
   python "%SCRIPT_DIR%\tools\detect_ci_matching_branch.py" "%ghprbSourceBranch%"
-  if "%ERRORLEVEL%" == "0" (
+  :: if error code is less than 1

```

Comment:
```suggestion
  :: if error code is not 1

```

Reply:
the syntax is a bit confusing. `if ERRORLEVEL N` does not use an equality test; it tests for `>= N`. So in order to test for a successful command (ie. return code == 0), we have to check for `!(return code >= 1)`

any recommendations on how to phrase that better?

Reply:
Ah ok, I just misunderstood it, I think the comment is good then

Reply:
I clarified the comment in https://github.com/ignition-tooling/release-tools/pull/618/commits/31111208f3f8e0c865cccdc9b7133cf26cb1ee42 anyway